README.md

```diff
@@ -76,7 +76,7 @@ The table below provides an overview of input/output data.
 |      [FASTREAD](#fastread)      |                    CSV                    |             :white_check_mark:(S)             |             CSV             |     :white_check_mark:      |     :white_check_mark:      |             :x:             |
 |        [Rayyan](#rayyan)        |    RIS, ENW, BIB, CSV, XML, CIW, NBIB     |             :white_check_mark:(M)             |     RIS, BIB, ENW, CSV      |     :white_check_mark:      |             :x:             |     :white_check_mark:      |
 |  [RobotAnalyst](#robotanalyst)  |                 RIS, NBIB                 | :white_check_mark::grey_question:<sup>5</sup> | :grey_question:<sup>5</sup> |     :white_check_mark:      | :grey_question:<sup>5</sup> |             :x:             |
-|  [SWIFT-Review](#swift-review)  |   RIS, TXT<sup>1</sup>, CSV, XML, NBIB    |             :white_check_mark:(M)             |             TXT             |     :white_check_mark:      |     :white_check_mark:      |             :x:             |
+|  [SWIFT-Active Screener](#swift-activescreener)  |   TXT, RIS, XML, BibTex    |             :white_check_mark:(M)             |             CSV, RIS             |     :white_check_mark:      |     :grey_question:      |             :white_check_mark:             |
```

Comment:
We should add a footnote to [this issue](https://github.com/Rensvandeschoot/software-overview-machine-learning-for-screening-text/issues/40) for the question mark about the re-import. I will add it to the issue (and rename the issue to SWIFT-Active Screener)

Reply:
solved in https://github.com/Rensvandeschoot/software-overview-machine-learning-for-screening-text/pull/53